nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/operations/MoveToColdStorageTest.java

```diff
@@ -21,15 +21,12 @@
 
 package org.nuxeo.coldstorage.operations;
 
-import static javax.servlet.http.HttpServletResponse.SC_CONFLICT;
-import static javax.servlet.http.HttpServletResponse.SC_FORBIDDEN;
-import static javax.servlet.http.HttpServletResponse.SC_NOT_FOUND;
+import static javax.servlet.http.HttpServletResponse.*;
```

Comment:
You need to have a look at https://doc.nuxeo.com/corg/intellij-idea-setup/#configure-nuxeo-code-style. We don't use the `*` could you configure you IDEA and reformat the files